packages/frameworks/src/frameworks.ts

```diff
@@ -1932,6 +1932,32 @@ export const frameworks = [
       },
     ],
   },
+  {
+    name: 'Storybook',
+    slug: 'storybook',
+    logo: 'https://api-frameworks.vercel.sh/framework-logos/storybook.svg',
+    tagline: 'Frontend workshop for UI development',
+    description:
+      'Storybook is a frontend workshop for building UI components and pages in isolation.',
+    website: 'https://storybook.js.org',
+    ignoreRuntimes: ['@vercel/next', '@vercel/node'],
+    settings: {
+      installCommand: {
+        placeholder: '`yarn install`, `pnpm install`, or `npm install`',
+      },
+      buildCommand: {
+        placeholder:
+          '`npm run build-storybook`, `yarn build-storybook`, or `pnpm build-storybook`',
+        value: 'npm run build-storybook',
+      },
+      devCommand: {
+        value: `start-storybook -p 6006`,
```

Comment:
This should use a PORT env var (see other frameworks)

Reply:
Done!